backend/__init__.py

```diff
@@ -17,8 +17,9 @@ def create_app(config=Config):
     db.init_app(app)
     login.init_app(app)
     Migrate(app, db)
-
+    
     # register blueprints
     app.register_blueprint(auth)
+    app.register_blueprint(api.blueprint)
```

Comment:
I think we should use `prefix = '/api' here

Reply:
Done